trview.ui/IFontMeasurer.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <trview.common/Size.h>
+#include <string>
+
+namespace trview
+{
+    namespace ui
+    {
+        struct IFontMeasurer
```

Comment:
Probably a comment about what the interface does and why it is necessary (the backwards link between the ui and the renderer)